src/Uno.UI.Composition/Composition/Visual.skia.cs

```diff
@@ -12,15 +14,39 @@ namespace Microsoft.UI.Composition;
 
 public partial class Visual : global::Microsoft.UI.Composition.CompositionObject
 {
+	// Since painting (and recording) is done on the UI thread, we need a single SKPictureRecorder per UI thread.
+	// If we move to a UI-thread-per-window model, then we need multiple recorders.
+	[ThreadStatic] private static SKPictureRecorder? _recorder;
```

Comment:
The more common formatting is attribute on separate line.

```suggestion
	[ThreadStatic]
	private static SKPictureRecorder? _recorder;
```